frontend/public/components/droppable-edit-yaml.tsx

```diff
@@ -42,10 +44,18 @@ export const DroppableEditYAML = withDragDropContext(class DroppableEditYAML ext
     if (file.size <= maxFileUploadSize) {
       const reader = new FileReader();
       reader.onload = () => {
-        const input = reader.result;
-        this.setState({
-          fileUpload: input,
-        });
+        const input:string = reader.result as string;
+        // eslint-disable-next-line no-control-regex
+        if (/[\x00-\x09\x0E-\x1F]/.test(input)) {
```

Comment:
Curious what this regex means?

Reply:
@benjaminapetersen Return true if a string contains non-printable characters. More here: http://stackoverflow.com/questions/1677644/detect-non-printable-characters-in-javascript

Reply:
Care to wrap in a named func?

```JavaScript 
if(nonPrintableCharacters(input) { 
``` 
Could make that clearer later.

Reply:
@benjaminapetersen added named function for non printable chars.

Reply:
Awesome.